### ISSUE TITLE
Issues with Reproducing pi0-SF on Real Hardware

### ISSUE THREAD

**Issue by CortexSphere** [OpenHelix-Team/Spatial-Forcing#6]
Previously, I successfully trained and deployed pi0.5-SF on the simulation dataset LIBERO PLUS, improving the success rate from 61.1% for pi0.5 to 75.2%. Now, when attempting to deploy this algorithm on real hardware using the default simulation parameters, I encounter situations where tasks that pi0.5 can handle perfectly result in a 0% success rate for pi0.5-SF. I would like to know if you and your team have any suggestions on this, such as parameter selection, dataset scale, etc.?

**Comment by ZeyuanHe** [comment]
Hi @CortexSphere, thank you so much for your LIBERO PLUS contribution! If convenient, would you like to share the corresponding loss curve on LIBERO PLUS for reference? Thank you so much!

**Comment by ZeyuanHe** [comment]
Hi @CortexSphere I wonder if you train pi0.5-sf on libero-plus data or the original libero data and get an libero-plus SR of 75%? I trained pi0.5-sf on original libero data and tested on libero-10 subset of libero-plus bug get an SR of 23% which is much lower than pi0.5-libero (65%).
Thank you so much!

**Comment by CortexSphere** [comment]
Hi @ZeyuanHe 

To clarify a few details about the setup: I actually trained the model using the LIBERO PLUS training set, rather than the original LIBERO dataset.

Additionally, for the evaluation, I randomly sampled about 1/10 of the original test set across four suites: goal, object, spatial, and 10. I tried my best to cover all the task variations in this subset, as running the complete evaluation is extremely time-consuming (though I might run the full test in the future).

Based on this setup, the final results I obtained are:

Original pi0.5 Success Rate (SR): ~61.1%

pi0.5-sf Success Rate (SR): ~75.2%
